src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormHelperText,
+  IconButton,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  TableCell,
+  TableRow,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import CancelIcon from '@mui/icons-material/Cancel';
+import prettyBytes from 'pretty-bytes';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AssociatedFile, LicenseNames, licenses } from '../../../types/associatedArtifact.types';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { Field, FieldProps, ErrorMessage, useFormikContext } from 'formik';
+import { SpecificFileFieldNames } from '../../../types/publicationFieldNames';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { DatePicker } from '@mui/x-date-pickers';
+import { TruncatableTypography } from '../../../components/TruncatableTypography';
+
+interface FilesTableRowProps {
+  file: AssociatedFile;
+  removeFile: () => void;
+  toggleLicenseModal: () => void;
+  baseFieldName: string;
+}
+
+export const FilesTableRow = ({ file, removeFile, toggleLicenseModal, baseFieldName }: FilesTableRowProps) => {
+  const { t } = useTranslation();
+  const [openConfirmDialog, setOpenConfirmDialog] = useState(false);
+  const toggleOpenConfirmDialog = () => setOpenConfirmDialog(!openConfirmDialog);
+  const { setFieldValue, setFieldTouched } = useFormikContext();
+
+  return (
+    <TableRow data-testid={dataTestId.registrationWizard.files.fileRow}>
+      <TableCell sx={{ wordBreak: 'break-word' }}>
+        <TruncatableTypography>{file.name}</TruncatableTypography>
+      </TableCell>
+
+      <TableCell align="center">
+        <Tooltip title={t('common.delete')}>
```

Comment:
Fra et UU-perspektiv (universell utforming) tror jeg denne burde være "Fjern fil" som før, i stedet for "Slett". Da er det tydeligere hva knappen gjør